lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -172,6 +180,8 @@ class RadarChartData extends BaseChartData with EquatableMixin {
             tickBorderData ?? const BorderSide(color: Colors.black, width: 2),
         gridBorderData =
             gridBorderData ?? const BorderSide(color: Colors.black, width: 2),
+        maxValue = maxValue,
```

Comment:
It's better to keep the maxValue non-nullable (It can be nullable in the constructor, and we can fill the non-nullable maxValue field using this logic: `(data.maxValue ?? data.maxEntry.value);`

Reply:
Check the below logic that we have implemented in the LineChartData:

https://github.com/imaNNeoFighT/fl_chart/blob/becd7bda3c9ff6fcc280c78763b438057bc2fa74/lib/src/chart/line_chart/line_chart_data.dart#L92-L97

Reply:
Thanks for the feedback. The new commits should implement these suggestions.